lib/generators/solidus_frontend/install/install_generator.rb

```diff
@@ -13,6 +13,19 @@ def copy_initializer
         template 'initializer.rb', 'config/initializers/solidus_frontend.rb'
       end
 
+      def robots_directives
+        append_file "public/robots.txt", <<-ROBOTS.strip_heredoc
+          User-agent: *
+          Disallow: /checkout
+          Disallow: /cart
+          Disallow: /orders
+          Disallow: /user
+          Disallow: /account
+          Disallow: /api
```

Comment:
This one looks a bit out of context, but because this is not something critical, I think we can live with a simple solution (keeping it).

Reply:
I ran a quick experiment with the tools available in generators/thor and couldn't find a reliable way to allow each part contribute it's paths to robots. I was unable to determine if we actually need to disallow api endpoints, in any case we could use a header for API instead https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#xrobotstag